src/controllers/UsersController.php

```diff
@@ -1533,6 +1533,7 @@ public function actionUploadUserPhoto(): ?Response
 
             return $this->asJson([
                 'html' => $this->_renderPhotoTemplate($user),
+                'assetId' => $user->photoId,
```

Comment:
Shouldn’t it be `photoId` to be consistent with the property?

Reply:
yes, yes it shuold